### PR TITLE
Added scary text

### PR DESCRIPTION
**Pull request by Thomas-Malcolm**
"Added" "scary" "text"

### CODE REVIEW

File: uqcsbot/text.py [hunk 1]
```diff
@@ -80,6 +80,14 @@ async def mock(self, ctx: commands.Context, *text: str):
         """
         await ctx.send("".join(choice((c.upper(), c.lower())) for c in " ".join(text)))
 
+    @commands.command()
+    async def scare(self, ctx: commands.Context, *text: str):
+        """
+        "adds" "scary" "quotes" "around" "the" "provided" "text"
+        """
+
+        await ctx.send(" ".join(f'"{w}"' for w in text.split(' ')))
```
Comment (TomCranitch) [inline uqcsbot/text.py:89]:
```suggestion
        await ctx.send(" ".join(f'"{w}"' for w in text))
```

**Review by TomCranitch — CHANGES_REQUESTED** [review]

File: uqcsbot/text.py (hunk 1 above)
Comment (kentonlam) [reply uqcsbot/text.py]: won't this just put quotes around every character?

Comment (kentonlam) [reply uqcsbot/text.py]: nvm

Comment (TomCranitch) [reply uqcsbot/text.py]: can we make !extrascare that does that pls

**Pull request merged**